test/plugin/scenarios/armeria-0.96plus-scenario/support-version.list

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+0.97.0
```

Comment:
@minwoox @trustin Once this PR merged, every time when Armeria has a new release, especially big release, such as 1.0, you could submit a pull request to add the version number here. Then the CI will verify whether it is still supported by SkyWalking ecosystem. We could merge that pull request and checked in any other further changes in the SkyWalking side.

Reply:
Thanks for information. Will try!